### DISCUSSION TITLE
Emoji Pops: A fun game built with VanJS under 60 lines

### DISCUSSION THREAD

**Discussion by Tao-VanJS** [vanjs-org/van#174] [Show and tell]
Hi fellow VanJSers,

Recently I built a fun game with **VanJS** and **VanX** under 60 lines: https://vanjs.org/demo#game

❤️ Hope you can enjoy!

Feel free to share your high scores in this thread :-)

**Comment by Tao-VanJS** [comment]
My current best is 125 🙂:

![Screenshot_20231103-221459](https://github.com/vanjs-org/van/assets/132854319/6ea96e11-b632-404b-b014-ebf0205cd429)

**Reply by mdovn** [reply]
mine is 45. lol

**Reply by sxxeun09** [reply]
~almost~ 200!
